### PR TITLE
CRM: Fix for Multiline items in the top-right menu

### PR DESCRIPTION
**Pull request by gogdzl**
## Proposed changes:
* This PR enhances the alignment of the top right menu items. Previously, when the text was too long, it would wrap under the icon, but now it respects the alignment and wraps appropriately within its own space.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
https://github.com/Automattic/zero-bs-crm/issues/3152

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:

* Change the language to spanish

In `trunk` you should get something like this:
image

In `fix/crm/3152-multiline-names-in-menu` you should see the items alignment as the screenshot below:
image

### CODE REVIEW

File: projects/plugins/crm/includes/ZeroBSCRM.Core.Menus.Top.php [hunk 1]
```diff
@@ -269,6 +269,8 @@ function zeroBSCRM_admin_top_menu( $branding = 'zero-bs-crm', $page = 'dash' ) {
 			});
 		})
 		</script>
+		<style>
+		</style>
```
Comment (cleacos) [inline projects/plugins/crm/includes/ZeroBSCRM.Core.Menus.Top.php]: Do we need this empty style element?

Comment (cleacos) [inline projects/plugins/crm/includes/ZeroBSCRM.Core.Menus.Top.php]:
```suggestion
```

**Review by cleacos — DISMISSED** [review]
It works as expected. I checked all the links too.

image

I left a minor comment to remove an empty <style> tag. Other than that, LGTM!

File: projects/plugins/crm/includes/ZeroBSCRM.Core.Menus.Top.php (hunk 1 above)
Comment (gogdzl) [reply projects/plugins/crm/includes/ZeroBSCRM.Core.Menus.Top.php]: Leftover from tests! Thanks for pointing that out!

**Review by cleacos — APPROVED** [review]
re-LGTM!

**Pull request merged**